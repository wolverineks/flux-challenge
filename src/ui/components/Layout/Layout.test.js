// @flow

import React from 'react'
import renderer from 'react-test-renderer'
import {Layout} from '.'

describe('Layout', () => {
  test('Layout', () => {
    const component = renderer.create(<Layout />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
