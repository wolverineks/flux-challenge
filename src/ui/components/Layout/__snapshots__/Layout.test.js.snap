// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Layout 1`] = `
<div
  className="app-container"
>
  <div
    className="css-root"
  />
</div>
`;
